graphql/e2e/common/common.go

```diff
@@ -54,6 +54,7 @@ var (
 	dgraphStateURL  = "http://" + Alpha1HTTP + "/state"
 
 	// this port is used on the host machine to spin up a test HTTP server
+	//nolint:unused
 	lambdaHookServerAddr = ":8888"
```

Comment:
It is used in lambda.go:376, so there is no need to add `nolint:unused`

Reply:
+1 ^ (for some reason when I ran the golangci locally, it flagged a failure for this). I noticed this was being used in another file - which is why I added a `nolint`

But my re-test did not flag a failure here. So lets get your PR in .. thanks for cross checking!